components/Scanner/index.js

```diff
@@ -0,0 +1,113 @@
+//copied from https://github.com/ericblade/quagga2-react-example/blob/master/src/components/Scanner.js
+import { useCallback, useLayoutEffect } from "react";
+import Quagga from "@ericblade/quagga2";
+
+function getMedian(arr) {
+  arr.sort((a, b) => a - b);
```

Comment:
Please use a copy off the array here to apply the sort method. We never want to manipulate a functions argument:
```
const myArray = [...arr].sort((a, b) => a - b)
```

Reply:
done